ac.soton.coda.simulator.feature/feature.xml

```diff
@@ -28,7 +40,7 @@
       <import plugin="org.eclipse.emf.ecore" version="2.7.0" match="compatible"/>
       <import plugin="org.eclipse.emf.transaction" version="1.4.0" match="compatible"/>
       <import plugin="org.eclipse.emf.workspace" version="1.5.0" match="compatible"/>
-      <import plugin="org.eclipse.emf.ecore.xmi" version="2.10.1" match="equivalent"/>
+      <import plugin="org.eclipse.emf.ecore.xmi" version="2.12.0" match="compatible"/>
```

Comment:
This will force users to update from 2.10 to 2.12.  Probably a good thing?

Reply:
I believe that 2.12 is the version shipped with Eclipse Neon.